python3Packages.django_extensions tests fail

This breaks the build when the mailman module is enabled.

```
$ nix-build -A python3Packages.django_extensions
[...]
=========================== short test summary info ============================
FAILED tests/templatetags/test_syntax_color.py::SyntaxColorTagTests::test_should_colorize_table_with_default_lexer
= 1 failed, 467 passed, 4 skipped, 3 xfailed, 7 xpassed, 237 warnings in 20.79s =

```

Caused by changes in https://github.com/NixOS/nixpkgs/pull/117452

--- a/pkgs/development/python-modules/django-extensions/default.nix
+++ b/pkgs/development/python-modules/django-extensions/default.nix
@@ -18,13 +18,13 @@
 
 buildPythonPackage rec {
   pname = "django-extensions";
-  version = "3.1.1";
+  version = "3.1.3";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = version;
-    sha256 = "0ss5x3d21c3g8i1s79l4akazlf116yp4y50gx4vrk1dxh3jb29zj";
+    sha256 = "03mhikhh49z8bxajbjf1j790b9c9vl4zf4f86iwz7g0zrd7jqlvm";
   };
 
   LC_ALL = "en_US.UTF-8";

